kgforge/core/archetypes/read_only_store.py

```diff
@@ -54,9 +54,19 @@ def __init__(
     def __repr__(self) -> str:
         return repr_class(self)
 
-    @abstractmethod
+    @staticmethod
+    def _context_to_dict(c: Context):
```

Comment:
please extend the c to be context, same for k and v to key and value, single character variables ... I dunno ...